catalog/services/enricher.go

```diff
@@ -0,0 +1,72 @@
+package services
+
+import (
+	"bytes"
+	"errors"
+	"io"
+	"net/http"
+
+	"github.com/CycloneDX/cyclonedx-go"
+	"github.com/jfrog/jfrog-client-go/auth"
+	"github.com/jfrog/jfrog-client-go/http/jfroghttpclient"
+	"github.com/jfrog/jfrog-client-go/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+)
+
+const (
+	catalogEnrichApi              = "api/v1/beta/cyclonedx/enrich"
+	catalogMinVersionForEnrichApi = "1.0.0"
```

Comment:
nit: where is this being used?

Reply:
In our security-cli repository while running the scan commands, used here:
https://github.com/jfrog/jfrog-cli-security/pull/478